pkg/addon/addon.go

```diff
@@ -1069,6 +1070,16 @@ func (h *Installer) dispatchAddonResource(addon *InstallPackage) error {
 		return errors.Wrap(err, "render addon definitions fail")
 	}
 
+	if !h.overrideDefs {
+		existDefs, err := checkConflictDefs(h.ctx, h.cli, defs, *app)
+		if err != nil {
+			return err
+		}
+		if len(existDefs) != 0 {
+			return fmt.Errorf(`definitions: %s in this addon already exist, if you want override them, Please use add "--override-defs" to re-enable or re-upgrade`, existDefs)
```

Comment:
```suggestion
			return fmt.Errorf(`definitions: %s in this addon already exist, if you want override them, Please use arguments "--override" to enable`, existDefs)
```

I wonder if the case really exist when users are upgrading addons?

Reply:
Shall we update an addon with new definitions add?

Reply:
I think addon developer cannot avoid this situation in the future.

Reply:
can we notify what exsiting addon contain this definition? It's user friendly:

···
definition: %s in this addon already exist in %s ...
···

Reply:
Yes, when conflict, will produce an error to tell what defs has exist.

Reply:
The print result is 
```
definition: helm already exist and not belong to any addon 
definition: kustomize in this addon already exist in fluxcd 
if you want override them, Please use arguments '--override' to enable